### ISSUE TITLE
CPU halts at booting up to install update 10.14.4

### ISSUE THREAD

**Issue by mikedebian** [AMD-OSX/AMD_Vanilla#3]
Previous updates work fine, but when trying to update to 10.14.4 from 10.14.3, the CPU halts completely after:

End Random Seed
++++++++++++++++++++++

No kernel panic, no log, just a complete halt before it boots up to update.

Can reboot back in to 10.14.3.

Ryzen 1800X
Prime X370-Pro
Vega 56

Done both clover stable and clover beta, updated kexts.

**Comment by dhinakg** [comment]
These patches do not support 10.14.4 yet.

**Comment by tvhung83** [comment]
Can anyone confirm that it works with 10.14.4 installer? Mine stopped at the same logs.

**Comment by mikigal** [comment]
@tvhung83 yep, latest patch version support 10.14.4. Check do u have updated patch and Nullcpupowermanagement. Show your logs.

**Comment by tvhung83** [comment]
@mikigal weird, I used the latest patch.

At first, I just copied the patch to my `config.plist`. But on 2nd attempt, I even cloned almost the whole `config.plist` (except SMBIOS part), still stopped at this line:
```
End RandomSeed
++++++++++++++++++++++
```
And yes, I always have `NullCPUPowerManagement.kext` in `Other`.

Btw, I'm booting to 10.14.4 installer (USB installer), not to an installed partition. I guess it doesn't matter, but just in case.

**Comment by mikigal** [comment]
Do u have OsxAptioFixDrv?

**Comment by tvhung83** [comment]
@mikigal nope, let me try it real quick and report back. Thank you for your responses!

**Comment by tvhung83** [comment]
Nope, still no luck (I remembered that I didn't need it to boot with 10.14.3 patch).
Please find attached my `config.plist`, and list of drivers & kexts:
```
❯ l /Volumes/EFI/EFI/CLOVER/kexts/Other
total 19K
drwxrwxrwx 1 robert staff  512 Mar 21 07:20 AppleALC.kext
drwxrwxrwx 1 robert staff  512 Nov 13  2017 FakeSMC.kext
drwxrwxrwx 1 robert staff  512 Mar 21 07:13 Lilu.kext

[config.plist.zip](https://github.com/AMD-OSX/AMD_Vanilla/files/3053543/config.plist.zip)
drwxrwxrwx 1 robert staff  512 Jan 21 00:19 NullCPUPowerManagement.kext
drwxrwxrwx 1 robert staff  512 Jan 21 00:19 RealtekRTL8111.kext
drwxrwxrwx 1 robert staff  512 Mar 21 07:22 WhateverGreen.kext

~
❯ l /Volumes/EFI/EFI/CLOVER/drivers64UEFI
total 277K
-rwxrwxrwx 1 robert staff  12K Mar 26 23:05 ApfsDriverLoader-64.efi
-rwxrwxrwx 1 robert staff  17K Mar 26 23:05 AppleImageCodec-64.efi
-rwxrwxrwx 1 robert staff 5.3K Mar 26 23:05 AppleKeyAggregator-64.efi
-rwxrwxrwx 1 robert staff 5.3K Mar 26 23:05 AppleKeyFeeder-64.efi
-rwxrwxrwx 1 robert staff 2.4K Mar 26 23:05 AppleUITheme-64.efi
-rwxrwxrwx 1 robert staff  22K Mar 26 23:05 AptioInputFix-64.efi
-rwxrwxrwx 1 robert staff  36K Mar 26 23:05 AptioMemoryFix-64.efi
-rwxrwxrwx 1 robert staff  45K Mar 26 23:05 AudioDxe-64.efi
-rwxrwxrwx 1 robert staff 5.5K Mar 26 23:05 DataHubDxe-64.efi
-rwxrwxrwx 1 robert staff  22K Mar 26 23:05 FSInject-64.efi
-rwxrwxrwx 1 robert staff 7.1K Mar 26 23:05 FirmwareVolume-64.efi
-rwxrwxrwx 1 robert staff  25K Oct  9  2016 HFSPlus.efi
-rwxrwxrwx 1 robert staff  14K Mar 26 23:05 HashServiceFix-64.efi
-rwxrwxrwx 1 robert staff  32K Mar 26 23:05 NvmExpressDxe-64.efi
-rwxrwxrwx 1 robert staff  24K Mar 26 23:05 OsxAptioFixDrv-64.efi
-rwxrwxrwx 1 robert staff 4.3K Mar 26 23:05 SMCHelper-64.efi
```

**Comment by tvhung83** [comment]
[config.plist.zip](https://github.com/AMD-OSX/AMD_Vanilla/files/3053545/config.plist.zip)

**Comment by mikigal** [comment]
I haven't how to check your config now. It looks like kernel problem, try config from my repository. I haven't idea what is wrong, if still no luck with my config ask on amd-osx discord

**Comment by tvhung83** [comment]
@mikigal Thanks! will definitely check it out. Btw, it's no rush, the rig is mostly for side projects, just curious if it can boot to 10.14.4 installer.

**Comment by mikedebian** [comment]
I would like to confirm that for my part, both the update to 10.14.4 and installing 10.14.4 from scratch (installer downloaded image) works on my Ryzen system.

**Comment by tvhung83** [comment]
Guys, after changing from Ryzen 3 2200G to Ryzen 5 2600, and also a new motherboard, all without luck. And when I followed https://vanilla.amd-osx.com guide, I didn't keep my hope high (because my method of creating USB installer is more "vanilla", using `createinstallmedia` on Mac). To my surprise, the installer worked!! Along with my Vega 56 and ALC887 (both didn't work when I used patch kernel). I wish someone will update `README` to include the guide link, or I can PR later.
